### PR TITLE
arrow keys navigation added | fixes mmcgrana/gobyexample#222

### LINKED ISSUE #222: Add arrow key navigation?
Left/right arrow keys to go to previous/next page would be cool

### PR DESCRIPTION
**Pull request by Shivam010**
Hi,
I implemented the simple JS `onkeydown` event for each page on `Left` and `Right` arrow keys for corresponding previous/next page. I hope it can be accepted easily. 
Thank you

Fixes #222

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+mavis.ml
```
Comment (mmcgrana) [inline CNAME]: Not sure what this file is for, please remove it for this PR.

File: templates/site.css [hunk 2]
```diff
@@ -75,7 +75,7 @@ div.example table {
     margin-top: 15px;
     margin-bottom: 20px;
 }
-p.next {
+p.next p.prev {
```
Comment (mmcgrana) [inline templates/site.css]: I don't think this CSS does what you expected 😅 Let's revert this change per the comment above about previous links.

File: templates/example.tmpl [hunk 3]
```diff
@@ -25,6 +39,11 @@
         {{end}}
       </table>
       {{end}}
+      {{if .PrevExample}}
+      <p class="prev">
```
Comment (mmcgrana) [inline templates/example.tmpl]: I don't want to add previous links right now, let's just introduce the arrow keys in this PR.

File: CNAME (hunk 1 above)
Comment (Shivam010) [reply CNAME]: Sorry for the CNAME file, I was experimenting something with my DNS server and Github custom domain feature, which results in all this

File: templates/site.css (hunk 2 above)
Comment (Shivam010) [reply templates/site.css]: Ok 😅

File: templates/example.tmpl (hunk 3 above)
Comment (Shivam010) [reply templates/example.tmpl]: Sure, no problem

**Comment by Shivam010** [comment]
@mmcgrana 
The changes are completed as requested. You can proceed forward.

**Comment by mmcgrana** [comment]
Great! I've rebased, squashed, and merged this. Thanks again!
